### PR TITLE
Re-order build tasks to enable publishing to central

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

### CODE REVIEW

**Review by aashikam — APPROVED** [review]
LGTM

**Review by ThisaruGuruge — APPROVED** [review]

File: mysql-ballerina/build.gradle [hunk 1]
```diff
@@ -247,6 +241,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push ${packageName}"
                 }
             }
+        } else {
+            println("Central Access Token is Not Present")
```
Comment (ThisaruGuruge) [inline mysql-ballerina/build.gradle:245]:
```suggestion
            throw new InvalidUserDataException("Central Access Token is Not Present")
```

File: mysql-ballerina/build.gradle [hunk 2]
```diff
@@ -247,6 +241,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push ${packageName}"
                 }
             }
+        } else {
```
Comment (ThisaruGuruge) [inline mysql-ballerina/build.gradle:244]: Shall we add the condition to check the `SNAPSHOT` version here?

Ref: https://github.com/ballerina-platform/module-ballerina-encoding/pull/80

Comment (niveathika) [reply mysql-ballerina/build.gradle:247]: Fixed in e15438d

**Pull request merged**